src/base-implementation.ts

```diff
@@ -0,0 +1,103 @@
+import { Wallet } from '@harmony-js/account'
+import { Contract } from '@harmony-js/contract'
+import { Harmony } from '@harmony-js/core'
+import { BigNumber, logger, constants } from 'ethers'
+import { Logger } from 'ethers/lib/utils'
```

Comment:
Why is logger twice?

Reply:
Idk, I didn't add those dependencies, I'll change it right away!